src/Test/Docs.hs

```diff
@@ -21,15 +21,14 @@ main = shaken noTest $ \args obj -> do
 
     config %> \_ -> do
         need ["shake.cabal"]
-        unit $ cmd "runhaskell Setup.hs configure" ["--builddir=" ++ obj "dist","--user"]
+        unit $ cmd "cabal configure" ["--builddir=" ++ obj "dist","--user"]
```

Comment:
Is moving to `cabal` required for running in a sandbox? I think I used to use `cabal` instead of `runhaskell`, but then it started not working and I switched. Generally `cabal` would be preferable if it always worked.